src/utils/security/password-hash.ts

```diff
@@ -0,0 +1,7 @@
+import { hash } from "bcrypt";
+
+export class HashProvider {
```

Comment:
I think you don't need separate class to generate hash. It can be just one simple function.